### ISSUE TITLE
Deprecated endpoints no longer strikethrough on sidebar

### ISSUE THREAD

**Issue by iqfareez** [scalar/scalar#1104]
### What happens?

Current behavior for deprecated endpoints 

![image](https://github.com/scalar/scalar/assets/60868965/4d9f4680-c3fd-4531-a941-9bd3c967d344)

It appears correctly on the title, just the sidebar. Is this intentional?

![image](https://github.com/scalar/scalar/assets/60868965/949ce72a-e8c6-4556-abfd-869719517e6b)

### What did you expect to happen?

The deprecated endpoints should be strikethrough.

![image](https://github.com/scalar/scalar/assets/60868965/33442b32-f165-4a34-8ec6-ab460fd3352c)

### How can we reproduce the issue?

I'm on Scalar latest version `1.17.15`

Mark endpoint as deprecated

```
get:
      ...
      summary: Prayer time for all zone
      deprecated: true
      ...
```

**Comment by marclave** [comment]
Thanks for reporting this! We did have it crossed out in the sidebar before, we must have made a regression by accident! 

@cameronrohani and I are meeting up tonight and can fix that :)

**Comment by hanspagel** [comment]
Addressed in #1116 👍
